scripts/projectView.js

```diff
@@ -1,87 +1,90 @@
-var projectView = {};
+(function(module){
+  var projectView = {};
 
-projectView.handleMainNav = function(){
-  $('.main-nav .tab').on('click', function(){
-    var $tabContent = $('.tab-content');
-    var $dataContent = $(this).attr('data-content');
-    $tabContent.hide();
-    $tabContent.filter('#' + $dataContent).fadeIn();
-  });
+  projectView.handleMainNav = function(){
+    $('.main-nav .tab').on('click', function(){
+      var $tabContent = $('.tab-content');
+      var $dataContent = $(this).attr('data-content');
```

Comment:
This variable is used only once, so you could just put the code on the right of the "=" straight into the .filter statement.